Lib/fontTools/merge/__init__.py

```diff
@@ -53,6 +53,13 @@ def __init__(self, options=None):
 
 		self.options = options
 
+	def _openFonts(self, fontfiles):
+		fonts = [ttLib.TTFont(fontfile) for fontfile in fontfiles]
+		for font,fontfile in zip(fonts, fontfiles):
+			font.fontfile = fontfile
+			font.name = font['name'].getDebugName(3)
```

Comment:
hm so you're using the TTFont inner `__dict__` to store additional attributes. It works but I'm not a huge fan, it kind of muddies the interface and risk inadvertently overwriting things (who knows maybe in the future one adds public attributes named like those and forgets about this piece of code?). If you still want to use the object's inner dict, can you prefix the attributes with some underscore or label them e.g. `_meger__fontfile` or something?